contracts/src/v0.8/dev/KeeperRegistryBase.sol

```diff
@@ -126,7 +126,7 @@ abstract contract KeeperRegistryBase is ConfirmedOwner, ExecutionPrevention, Ree
     uint96 balance;
```

Comment:
after this PR, we can rearrange this to

struct Upkeep {
    uint96 balance;
    address lastKeeper; // 1 full evm word
    uint96 amountSpent;
    address admin; // 2 full evm words
    uint32 executeGas;
    uint32 maxValidBlocknumber;
    address target;
    bool paused; // 24 bits to 3 full evm words
  }

Reply:
nit. can you include in this PR? In the current state the comments are wrong so better to fix together

Reply:
Done